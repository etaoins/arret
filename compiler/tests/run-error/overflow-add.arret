(import [stdlib base])
(import [stdlib test])

(def maximum-int 9223372036854775807)

(defn main! () ->! ()
  (black-box! (+ (black-box! maximum-int) 1))
  ())