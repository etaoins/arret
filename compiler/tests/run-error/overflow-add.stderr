attempt to add with overflow
